modules/raft-api/src/main/java/org/apache/ignite/internal/raft/RaftGroupConfiguration.java

```diff
@@ -32,11 +32,16 @@
 public class RaftGroupConfiguration implements Serializable {
```

Comment:
This class now seems to be identical to `CommittedConfiguration`. Do we need to keep both?

Reply:
removed